templates/build.gradle

```diff
@@ -30,7 +30,6 @@ ext {
     INJECT = {inject}
     ASSETS = {assets}
     NATIVES = {natives}
-    MERGED = {merged_src}
     INTELLIJ = Boolean.getBoolean('idea.active')
 }
 
```

Comment:
This stuff below is still causing symlinking problems in all versions that have an inject folder (which currently includes e.g. 1.21.8).